functions/go/starlark/starlark/processor.go

```diff
@@ -0,0 +1,37 @@
+package starlark
+
+import (
+	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
+)
+
+type Processor struct{}
+
+func (gkp *Processor) Process(resourceList *fn.ResourceList) error {
+	err := func() error {
+		sfc := &StarlarkFnConfig{}
```

Comment:
I found `StarlarkFnConfig.Transform` confusing because `Transform` is the transformer's main function, but `StarlarkFnConfig` more looks like the starlark functionConfig. 

Also, is that possible to use `ResourceListProcessorFunc` but not a custom `Processor`?

Reply:
I renamed `Transform` to `Evaluate`. Hope it makes more sense now.
I also updated it to use `ResourceListProcessorFunc`.

Reply:
sorry, I was suggesting to keep `Transform` but rename "StarlarkFnConfig" to something other than`FnConfig`. How about `StarlarkEvaludator` or `StarlarkFn`?